src/test/java/com/rybka/controller/BankControllerTest.java

```diff
@@ -0,0 +1,69 @@
+package com.rybka.controller;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.rybka.model.BankData;
+import com.rybka.service.exchange.BankAggregatorService;
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.web.servlet.MockMvc;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Random;
+
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static org.springframework.test.web.servlet.setup.MockMvcBuilders.standaloneSetup;
+
+@WebMvcTest(BankController.class)
+class BankControllerTest {
+
+    private MockMvc mockMvc;
```

Comment:
Try to place Autowire anotation and remove builder in the setUp method

Reply:
and remove controller from there

Reply:
fixed